RELEASE.md

```diff
@@ -68,7 +68,17 @@ Please make sure you have these tools installed before starting this process:
 
 -   [ ] 8.  Test
 
-    Test the entire release.
+    -   Validate that your feature is working according to the requirements
+
+    -   Do exploratory testing around the functionalitys your feature impacted
```

Comment:
functionalitys -> functionalities

Reply:
Do some exploratory tests to find previously existing features that might have impacted by these changes